e2e/address_types.test.js

```diff
@@ -11,10 +11,12 @@ describe('Example', () => {
 
   it('loads my test wallet', async () => {
     // let newWalletButton  element(by.text('CREATE NEW WALLET (NEW SEED)'));
-    await element(by.id("loadingapp.restorewalletseed")).tap();
-    await element(by.id("seed.seedplaceholder")).typeText('lottery multiply patient simple ivory leisure swift square west despair beauty match crowd margin reject box always title photo remind word diet ecology badge');
-    await element(by.id("birthdayinput")).typeText('1985000');
-    await element(by.text("RESTORE WALLET")).tap();
+    await element(by.id('loadingapp.restorewalletseed')).tap();
+    await element(by.id('seed.seedplaceholder')).typeText(
```

Comment:
Nobody changed your test... as you can see: `typeText`... the same you put here. I did a NEW test, and in MY test I prefer to put `replaceText` because I have no patience to wait the test write the seed.

I don't know if you have any other concerns about this? I'm happy to answer.

Reply:
oh, replaceText is better! im happy for you to change my code when you can improve it

im looking into test fixtures so we can share more code